### ISSUE TITLE
[sysex librarian] MIDI file tempo is not adopted

### ISSUE THREAD

**Issue by gritd** [krevis/MIDIApps#62]
When loading a midi file into sysex librarian the set tempo of the file is not respected by sysex librarian. This can cause issues on midi files which have time critical sysex data dumps.

**Comment by krevis** [comment]
Right. SysEx Librarian doesn't remember the timing of the sysex events in the original MIDI file.

(It isn't just about the tempo; in fact the tempo is largely irrelevant. In a MIDI file it's typical for the sysex data to be split into arbitrary chunks of different sizes, all occurring at different times, which may or may not follow a pattern.)

If your particular MIDI device needs the timing to be perfect, you can play the MIDI file using any DAW that supports sysex (which is most of them).

**Comment by gritd** [comment]
Well you will be surprised how many DAWs support sysex, Ableton does not and I think neither does Reason. Sysex Librarian is the one go-to program on OSX to get this done really!

**Comment by krevis** [comment]
I was going to say, most DAWs except for Ableton Live. (Live 10 supposedly has some amount of sysex support, but I don't know if it handles this case.)

I know Logic and Reaper can both do it. Reaper has a free trial.

**Comment by gritd** [comment]
Although this is 6 years ago already, this question pops up with us times and times again so I'll post a DAW alternative on OSX for playing back .mid files at correct speed: http://www.methodred.net/midiplayerx/ - It's a bit overkill to buy logic for this simple task...
